pyvrp/tests/test_GeneticAlgorithm.py

```diff
@@ -92,17 +93,21 @@ def test_params_constructor_does_not_raise_when_arguments_valid(
 def test_best_solution_improves_with_more_iterations():
     data = read("data/RC208.txt", "solomon", "dimacs")
     rng = XorShift128(seed=42)
-    pen_manager = PenaltyManager(data.vehicle_capacity)
-    pop = Population(data, pen_manager, rng, bpd)
-    ls = LocalSearch(data, pen_manager, rng, compute_neighbours(data))
+    pm = PenaltyManager(data.vehicle_capacity)
 
-    node_op = Exchange10(data, pen_manager)
+    pop_params = PopulationParams()
+    init = make_random_solutions(pop_params.min_pop_size, pm, rng, data)
+    pop = Population(init, bpd, params=pop_params)
+
+    ls = LocalSearch(data, pm, rng, compute_neighbours(data))
+
+    node_op = Exchange10(data, pm)
     ls.add_node_operator(node_op)
```

Comment:
While we're here anyway:

```suggestion
    ls = LocalSearch(data, pm, rng, compute_neighbours(data))
    ls.add_node_operator(Exchange10(data, pm))
```